code-challenges.test.js

```diff
@@ -11,41 +15,97 @@
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
+
 // a) Create a test with expect statements for each of the variables provided.
 
 const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
 const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
+
+
+// Message for instructors:
+// I had to comment out lines 3-6 for my tests to pass. Im not sure if that was part of the asessment that we were supposed to figure out?
+
 // b) Create the function that makes the test pass.
 
 // Pseudo code:
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
+// const object1 = { number: 15 }
 // Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
+// const object2 = { number: 0 }
+// // Expected output: "0 is divisible by three"
+// const object3 = { number: -7 }
 // Expected output: "-7 is not divisible by three"
+describe("divisibleByThree", () => {
+    it("takes a object as an argument and  decides if the number inside it is evenly divisible by three or not.", () => {
+    const object1 = { number: 15 }
+    const object2 = { number: 0 }
+    const object3 = { number: -7 }
+        expect(divisibleByThree(object1.number)).toEqual(`15 is divisible by three`)
+        expect(divisibleByThree(object2.number)).toEqual(`0 is divisible by three`)
+        expect(divisibleByThree(object3.number)).toEqual(`-7 is not divisible by three`)
```

Comment:
The function is expecting an object as an argument. Here, you are passing in a number.